google-ads-examples/src/main/java/com/google/ads/googleads/examples/targeting/AddDemographicTargetingCriteria.java

```diff
@@ -98,22 +104,27 @@ public static void main(String[] args) {
    * @param customerId the client customer ID in which to create criterion.
    * @param adGroupId the ad group ID in which to create criterion.
    */
-  private void runExample(GoogleAdsClient googleAdsClient, long customerId, long adGroupId) {
+  private void runExample(
+      GoogleAdsClient googleAdsClient,
+      long customerId,
+      long adGroupId,
+      Double bidModifier) {
```

Comment:
Can make this a lowercase 'd' `double`, since you've made the parameter required